src/h5web/vis-packs/core/heatmap/ColorBar.module.css

```diff
@@ -1,6 +1,5 @@
 .colorBar {
   display: grid;
-  padding: 1rem;
```

Comment:
I removed the padding as it was better in the `DomainSlider` to have `ColorBar` directly against the bars.

As a consequence, I added wrappers with padding in the other components using `ColorBar`